### PR TITLE
Fixed feedback and added Polynomial constructors for simpler code

### PR DESCRIPTION
**Pull request by mfalt**

### CODE REVIEW

**Comment by mfalt** [comment]
Solves https://github.com/JuliaControl/ControlSystems.jl/issues/163

File: src/types/tf.jl [hunk 1]
```diff
@@ -69,3 +69,23 @@ function tf(var::AbstractString, Ts::Real)
     Ts == 0 && error("Ts must not be 0 for discrete time tf.")
     return tf([1, 0], [1], Ts)
 end
+
+## Constructors for polynomial inputs
+function tf(num::AbstractArray{PT}, den::AbstractArray{PT},  Ts::Real=0.0) where {T<:Number, PT <: Polynomials.Poly{T}}
+    ny, nu = size(num, 1), size(num, 2)
+    if (ny, nu) != (size(den, 1), size(den, 2))
```
Comment (baggepinnen) [inline src/types/tf.jl]: `(size(den, 1), size(den, 2)) == size(den) `?

File: src/types/tf.jl [hunk 2]
```diff
@@ -69,3 +69,23 @@ function tf(var::AbstractString, Ts::Real)
...
+        error("num and den dimensions must match")
+    end
+
+    matrix = Matrix{SisoRational{T}}(undef, ny, nu)
+    for o=1:ny
```
Comment (baggepinnen) [inline src/types/tf.jl]: This double loop can be replaced by one over eachindex

File: src/types/tf.jl (hunk 1 above)
Comment (mfalt) [reply src/types/tf.jl]: Not if den is vector

File: src/types/tf.jl (hunk 2 above)
Comment (mfalt) [reply src/types/tf.jl]: That doesn't have to result in a `Matrix` type

**Pull request merged**